calisphere/collection_views.py

```diff
@@ -307,7 +307,7 @@ def get_mosaic(self):
                 repositories.append(repository['name'])
 
         if self.index == 'es':
-            sort = ("sort_title.raw", "asc")
+            sort = ("sort_title", "asc")
         else:
             sort = ("sort_title", "asc")
```

Comment:
This if/else doesn't seem to be doing anything anymore. Furthermore, the variable `sort` is only used in on place, down on line 328 `"sort": sort`. What about removing the whole if/else construct and setting `"sort": ("sort_title", "asc")`